### DISCUSSION TITLE
How to update a value where the property name uses umlauts or German sz?

### DISCUSSION THREAD

**Discussion by framps** [zivillian/ism7mqtt#149] [General]
Neither 
```
mosquitto_pub -h 192.168.0.12 -t 'Wolf/192.168.0.156/DHK_BM-2_0x35/set/Normau\\u00DFentemperatur_Heizkurve/340014/value' -m "-12"
```
nor
```
mosquitto_pub -h 192.168.0.12 -t 'Wolf/192.168.0.156/DHK_BM-2_0x35/set/Normaußentemperatur_Heizkurve/340014/value' -m "-12"
```
works for me :cry:

**Comment by zivillian** [comment]
There should be no MQTT topic containing an `ß`. Have you tried `Wolf/192.168.0.156/DHK_BM-2_0x35/set/Normaussentemperatur_Heizkurve/340014/value`?
Just use the topic under which the current value is reported.
`340014` in the topic also looks like you have multiple parameters with the same name - you should clean up your parameter.json and remove the unused duplicates.

**Reply by zivillian** [reply]
All umlauts and special characters are escaped in https://github.com/zivillian/ism7mqtt/blob/29f9e43de8ef65711cad0c2a32905730e4cbf991/src/ism7mqtt/ISM7/Ism7Config.cs#L509-L523
